modules/developers-guide/pages/cli-reference/dfx-stop.adoc

```diff
@@ -1,7 +1,7 @@
 = dfx stop
 
 Use the `+dfx stop+` command to stop the {IC} processes that you currently have running on your local computer.
-In most cases, you run the {IC} locally so that you can deploy canisters and test your programs during development.
+In most cases, you run the {IC} locally so that you can deploy canisters and test your dapps during development.
```

Comment:
```suggestion
In most cases, you run the canister execution environment locally so that you can deploy canisters and test your dapps during development.
```